src/translator/translate.py

```diff
@@ -75,12 +93,18 @@ def translate_parser_output(
 
     if new_parser_output.html_data is not None:
         for block in new_parser_output.html_data.text_blocks:
-            block.text = translate_text(translate_client, block.text, target_language)
+            if all([should_translate_text(text) for text in block.text]):
+                block.text = translate_text(
+                    translate_client, block.text, target_language
+                )
```

Comment:
hmm, the behaviour of this could be weird. `block.text` is an array of strings, right? 

I think what we want to do here is to decide for each item whether to translate it or not, rather than deciding to not translate the whole array because one item in the array is a number.

Reply:
I'm pretty sure that the data structure of out text blocks is just that block.text is an array of length 1. 

But yes we can handle these individually I guess!

Reply:
Updated the functionality to run conditionally on each individual text block 👍